cves/kernel/CVE-2016-3137.yml

```diff
@@ -105,16 +104,16 @@ vcc_instructions: |
 
   Place any notes you would like to make in the notes field.
 vccs:
-- commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
-  note: Discovered automatically by archeogit.
+- commit:
+  note: 
 - commit: 194343d9364ea07c9f27c4505380a15a905e8a24
-  note: Discovered automatically by archeogit.
+  note: 'Did not directly alter vulnerable code but did major changes to other tangentally related features.'
 - commit: 5c1a0f418d8d985f3a62849bcac43fc5404cc592
-  note: Discovered automatically by archeogit.
+  note: 'Changes made directly to function of vulnerable code.'
 - commit: 80b6ca48321974a6566a1c9048ba34f60420bca6
-  note: Discovered automatically by archeogit.
+  note: 'System wide alterations from kmalloc to kzalloc.'
 - commit: 813a224fa5bff3e34309a6494e231d5ebfa0fb4e
-  note: Discovered automatically by archeogit.
+  note: 'Major changes to cypress systems.'
 upvotes_instructions: |
```

Comment:
I would give this a 5 on the upvotes. I find vulnerabilities that involve curating specific things interesting

Reply:
This is pretty cool, we don't see a lot of CVEs related to physical attacks. Solid 4